errors/page-without-valid-component.md

```diff
@@ -0,0 +1,9 @@
+# Page Without Valid React Component
+
+#### Why This Error Occurred
+
+While auto exporting a page a valid React Component wasn't found. This could mean you have a file in `pages` that does `export default { some: 'object' }` instead of a React Component.
```

Comment:
```suggestion
While auto exporting a page a valid React Component wasn't found. This could mean you have a file in `pages` that exports something that is not a React Component.
```